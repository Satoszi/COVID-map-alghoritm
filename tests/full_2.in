8 8 2 13
full_2
++++++++
++++++++
++++++++
++++++++
++++++++
++++++++
++++++++
++++++++