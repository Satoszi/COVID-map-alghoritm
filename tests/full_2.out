7 7
0 7
3 6
6 5
1 5
7 3
4 3
0 3
2 2
5 1
7 0
3 0
0 0
